site/docs/4.3/components/carousel.md

```diff
@@ -62,18 +62,44 @@ Adding in the previous and next controls:
       {% include icons/placeholder.svg width="800" height="400" class="bd-placeholder-img-lg d-block w-100" color="#333" background="#555" text="Third slide" %}
     </div>
   </div>
-  <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev">
+  <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev" title="Previous">
     <span class="carousel-control-prev-icon" aria-hidden="true"></span>
     <span class="sr-only">Previous</span>
   </a>
-  <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next">
+  <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next" title="Next">
     <span class="carousel-control-next-icon" aria-hidden="true"></span>
     <span class="sr-only">Next</span>
   </a>
+  <div class="carouselButton">
+    <button id="pauseCarouselButton" type="button" class="btn btn-default btn-xs carousel-button" aria-label="Pause Carousel" title="Pause Carousel">
```

Comment:
focus is not visible on that element, carousel-button class is defined in o-carousel which is dedicated to custom carousel.